addon/components/paper-button.js

```diff
@@ -4,45 +4,36 @@ import RippleMixin from '../mixins/ripple-mixin';
 import ProxiableMixin from 'ember-paper/mixins/proxiable-mixin';
 import ColorMixin from 'ember-paper/mixins/color-mixin';
 
+const {
+  assert,
+  computed,
+  isPresent
+} = Ember;
+
 export default BaseFocusable.extend(RippleMixin, ProxiableMixin, ColorMixin, {
-  attributeBindings: ['target', 'action', 'type'],
+  attributeBindings: ['type'],
   tagName: 'button',
+  classNames: ['paper-button', 'md-default-theme'],
   themed: true,
-  classNameBindings: ['raised:md-raised', 'icon-button:md-icon-button', 'focus:md-focused', 'themed:md-default-theme', 'themed:md-button'],
+  classNameBindings: [
+    'raised:md-raised',
+    'iconButton:md-icon-button',
+    'themed:md-default-theme',
+    'themed:md-button'
+  ],
 
   // Ripple Overrides
   rippleContainerSelector: null,
-  fitRipple: Ember.computed.readOnly('isIconButton'),
-  center: Ember.computed.readOnly('isIconButton'),
-  dimBackground: Ember.computed.not('isIconButton'),
-
-  noSpan: Ember.computed('no-span', function() {
-    return this.get('no-span');
-  }),
+  fitRipple: computed.readOnly('isIconButton'),
+  center: computed.readOnly('isIconButton'),
+  dimBackground: computed.not('isIconButton'),
 
-  // RippleMixin overrides
-  isIconButton: Ember.computed(function() {
-    return this.classNames.any(function(className) {
-      return className.indexOf('md-icon-button') !== -1;
-    });
-  }),
-  isMenuItem: Ember.computed(function() {
-    return this.classNames.any(function(className) {
-      return className.indexOf('md-menu-item') !== -1;
-    });
-  }),
+  noSpan: computed.readOnly('no-span'),
```

Comment:
I don't know were this came from, but let's never use a `<span`. Angular Material uses it because it has to do with the way it does bindings, I think.
Also remove from paper-button template.